uk.co.compucorp.civicrm.tasksassignments/js/test/mocks/fabricators/file.js

```diff
@@ -1,30 +1,32 @@
-define(function() {
```

Comment:
You added mocks in the previous commit, there is no need then to have semistandard in a separate commit, semistandard should have been run before commit. Please squash this commit with the previous one.